lib/elixir_analyzer/test_suite/name_badge.ex

```diff
@@ -0,0 +1,22 @@
+defmodule ElixirAnalyzer.TestSuite.NameBadge do
+  @moduledoc """
+  This is an exercise analyzer extension module for the concept exercise NameBadge
+  """
+
+  use ElixirAnalyzer.ExerciseTest
+  alias ElixirAnalyzer.Constants
+
+  feature "Use an if macro" do
+    find :one
+    type :essential
```

Comment:
This is another case where I think this is too strong of a suggestion.

```suggestion
    type :actionable
```

Reply:
But this is a learning exercise. I really think it should remain `essential`.

Reply:
I'll see on Slack what others think about this.